django_ibmi/schemaEditor.py

```diff
@@ -32,6 +32,9 @@
 
 
 class DB2SchemaEditor(BaseDatabaseSchemaEditor):
+    def quote_value(self, value):
+        pass
```

Comment:
Shouldn't this return something even if it's just `value`?

https://github.com/django/django/blob/76c0b32f826469320c59709d31e2f2126dd7c505/django/db/backends/mysql/schema.py#L52